include/swift/AST/LookupKinds.h

```diff
@@ -65,8 +65,14 @@ enum NLOptions : unsigned {
   /// primary source file.
   ///
   /// \see NL_KnownDependencyMask
-  NL_KnownNoDependency =
+
+  // Work around MSVC bug: expression did not evaluate to a constant.
```

Comment:
I can't seem to reproduce this - I think it is a Visual Studio bug. I've removed the workaround